content/en/synthetics/guide/api_test_timing_variations.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Understanding API test timings and troubleshooting their variations
+kind: documentation
+description: Understanding API test timings and troubleshooting their variations
+aliases:
+- /synthetics/api_test_timing_variations
+further_reading:
+- link: "https://docs.datadoghq.com/synthetics/metrics/#api-tests"
+  tag: "Documentation"
+  text: "Synthetics API test timing metrics"
+---
+
+
+## Overview
+
+Synthetics API tests collect several [**timing metrics**][1] which can be used to identify any bottlenecks in the communication between the client and your server. 
+
+## Collected Timings
+
+
+The metrics collected by Synthetics measure the following timings : 
+
+
+**1. Redirection time**
```

Comment:
What do you think of having ### for all these? And also for the ones in the variations section?